src/main/java/cubicchunks/server/chunkio/IONbtReader.java

```diff
@@ -177,7 +177,11 @@ private static void readEntities(NBTTagCompound nbt, ICubicWorldServer world, Cu
 					.getName(), entityCubeX, entityCubeY, entityCubeZ, cube.getX(), cube.getY(), cube.getZ()));
 			}
 
+			// Let the entity know what Cube it is in... Its in a cube, right? :P
```

Comment:
As I said in the comment, i would explain what would happen if you don't do that.

This comment doesn't tell you anything you don't see from code.